docs/core/docker/index.md

```diff
@@ -1,18 +1,4 @@
 ---
-title: Docker and .NET Core
-description: Learn about using Docker with .NET Core using these fun tutorials.
-author: jralexander
-ms.author: johalex
-ms.date: 01/10/2018
+redirect_url: intro-net-docker
```

Comment:
Something is wrong here. It's not redirecting on the review site. It's showing me the empty page.

Reply:
Could be because of the invalid yaml header. Hold on I'll fix that and retest